base/irrationals.jl

```diff
@@ -110,6 +110,8 @@ end
 @assume_effects :total function rationalize(::Type{T}, x::AbstractIrrational; tol::Real=0) where T
     return rationalize(T, big(x), tol=tol)
 end
+rationalize(x::AbstractIrrational; kwargs...) = rationalize(Int, x)
```

Comment:
For a more bold and general solution, we could change

```julia
rationalize(x::AbstractFloat; kvs...) = rationalize(Int, x; kvs...)
```
to
```julia
rationalize(x::Real; kvs...) = rationalize(Int, x; kvs...)
```
in rational.jl instead